app/javascript/components/Admin/SearchPopover.tsx

```diff
@@ -42,6 +42,7 @@ const SearchPopover = () => {
     user_query: initialQueries.user_query,
     purchase_query: initialQueries.purchase_query,
     affiliate_query: initialQueries.affiliate_query,
+    order_id: searchParams.get("order_id") || "",
```

Comment:
I don't think we need these changes? (since admin already supports searching by `query`)